backend/sample.env

```diff
@@ -97,6 +97,8 @@ STRUCTURE_TOOL_IMAGE_TAG="0.0.36"
 # Feature Flags
 EVALUATION_SERVER_IP=unstract-flipt
 EVALUATION_SERVER_PORT=9000
+EVALUATION_SERVER_WARNINGS="false"
```

Comment:
why this wrapped in double quotes. I believe it would still be treated as a string even without the quote